api/firewaller/application.go

```diff
@@ -36,6 +36,25 @@ func (s *Application) Watch() (watcher.NotifyWatcher, error) {
 	return common.Watch(s.st.facade, "Watch", s.tag)
 }
 
+// Life returns the application's current life state.
```

Comment:
This reverts a change I made on this branch, but I just back-ported it to 2.5 . I will shepherd it back in with a another merge :)